masterlist.yaml

```
@@ -17475,6 +17526,21 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/8387/' ]
     msg: [ *alreadyInLotD ]
 
+  - name: 'Thaumaturgy.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/57138/' ]
+    tag:
+      - EffectStats
+      - Enchantments
+      - EnchantmentStats
+      - Graphics
+      - Keywords
+      - Relev
+      - Stats
+      - Text
+  - name: 'ThaumaturgyJumpBoots.esp'
+    tag:
+      - EnchantmentStats
```

Single-entry sets should be written in YAML's flow style for stylistic consistency:
```suggestion
    tag: [ EnchantmentStats ]
```